### PR TITLE
Change safegraph parameter hierarchy

### PR DESCRIPTION
**Pull request by sgsmob**
### Description
Update the safegraph `params.json` file and loading code to follow the conventions of #795

### CODE REVIEW

File: safegraph/params.json.template [hunk 1]
```diff
@@ -1,17 +1,23 @@
...
-  "aws_secret_access_key": "",
-  "aws_default_region": "",
-  "aws_endpoint": "",
-  "sync": true,
-  "wip_signal" : ["median_home_dwell_time_7dav",
-                  "completely_home_prop_7dav",
-                  "part_time_work_prop_7dav",
-                  "full_time_work_prop_7dav"],
-  "log_exceptions": false
+  "common": {
```
Comment (chinandrew) [inline safegraph/params.json.template:2]: in #795 , you use `shared` instead of `common`, is that change intentional? 

Also, is export dir  here isntead of `indicator` because it's also shared with the archive step?

**Comment by chinandrew** [comment]
this also needs to be updated https://github.com/cmu-delphi/covidcast-indicators/blob/main/ansible/templates/safegraph-params-prod.json.j2. I wonder if there's a better way to keep these in sync... I've definitely forgotten it before

File: safegraph/params.json.template (hunk 1 above)
Comment (sgsmob) [reply safegraph/params.json.template:2]: No they should both be the same.  I just changed #795 to be `common` although I can switch them all to `shared` instead too.

Export dir is a parameter shared in the validation step too.

**Comment by sgsmob** [comment]
> this also needs to be updated https://github.com/cmu-delphi/covidcast-indicators/blob/main/ansible/templates/safegraph-params-prod.json.j2. I wonder if there's a better way to keep these in sync... I've definitely forgotten it before

Thanks, done

File: safegraph/params.json.template (hunk 1 above)
Comment (chinandrew) [reply safegraph/params.json.template:2]: oh right, because you need to validate the output :facepalm:. `common` sounds good to me.

**Review by chinandrew — APPROVED** [review]
:shipit::shipit::shipit:

File: ansible/templates/safegraph-params-prod.json.j2 [hunk 2]
```diff
@@ -1,14 +1,20 @@
...
+  "indicator": {
+    "static_file_dir": "./static",
+    "raw_data_dir": "/common/safegraph",
+    "n_core": "12",
+    "aws_access_key_id": "{{ safegraph_aws_access_key_id }}",
+    "aws_secret_access_key": "{{ safegraph_aws_secret_access_key }}",
+    "aws_default_region": "us-east-1",
+    "aws_endpoint": "https://s3.wasabisys.com",
+    "sync": true,
+    "wip_signal" : [],
```
Comment (chinandrew) [inline ansible/templates/safegraph-params-prod.json.j2:15]:
```suggestion
    "wip_signal" : []
```

Comment (chinandrew) [reply ansible/templates/safegraph-params-prod.json.j2:15]: caught by json ci check

Comment (sgsmob) [reply ansible/templates/safegraph-params-prod.json.j2]: thanks
